.gitlab-ci.yml

```diff
@@ -49,6 +57,7 @@ deploy-dev-iowa:
 dev-iowa-tests:
   extends:
     - .dev-iowa
+    - .dev-gcp-migration
```

Comment:
I think this just overwrites the values from `.dev-iowa` rather than running against both instances. I think you'd have to create a new `dev-gcp-migration-tests` section.

Reply:
ah okay, let me try that